commandline/WSL/install_guide.md

```diff
@@ -1,84 +1,93 @@
 ---
-title: Bash on Ubuntu on Windows - Installation Guide
-description: Installation Guide
-keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu
-author: jackchammons
-ms.date: 9/6/2016
+title: Install Linux Subsystem on Windows 10

```

Comment:
Agreed.  I remember there was a big conversation around which term to use but now that the bash part and Ubuntu part are through the store, it makes sense as a systemic change.